rust/private/rustc_resource_set.bzl

```diff
@@ -0,0 +1,296 @@
+    return {"cpu": 23}
+
+def _resource_set_cpu_24(_os_name, _inputs):
+    return {"cpu": 24}
+
+def _resource_set_cpu_25(_os_name, _inputs):
+    return {"cpu": 25}
+
+def _resource_set_cpu_26(_os_name, _inputs):
+    return {"cpu": 26}
+
+def _resource_set_cpu_27(_os_name, _inputs):
+    return {"cpu": 27}
+
+def _resource_set_cpu_28(_os_name, _inputs):
+    return {"cpu": 28}
+
+def _resource_set_cpu_29(_os_name, _inputs):
+    return {"cpu": 29}
+
+def _resource_set_cpu_30(_os_name, _inputs):
+    return {"cpu": 30}
+
+def _resource_set_cpu_31(_os_name, _inputs):
+    return {"cpu": 31}
+
+def _resource_set_cpu_32(_os_name, _inputs):
+    return {"cpu": 32}
+
+def _resource_set_cpu_33(_os_name, _inputs):
+    return {"cpu": 33}
+
+def _resource_set_cpu_34(_os_name, _inputs):
+    return {"cpu": 34}
+
+def _resource_set_cpu_35(_os_name, _inputs):
+    return {"cpu": 35}
+
+def _resource_set_cpu_36(_os_name, _inputs):
+    return {"cpu": 36}
+
+def _resource_set_cpu_37(_os_name, _inputs):
+    return {"cpu": 37}
+
+def _resource_set_cpu_38(_os_name, _inputs):
+    return {"cpu": 38}
+
+def _resource_set_cpu_39(_os_name, _inputs):
+    return {"cpu": 39}
+
+def _resource_set_cpu_40(_os_name, _inputs):
+    return {"cpu": 40}
+
+def _resource_set_cpu_41(_os_name, _inputs):
+    return {"cpu": 41}
+
+def _resource_set_cpu_42(_os_name, _inputs):
+    return {"cpu": 42}
+
+def _resource_set_cpu_43(_os_name, _inputs):
+    return {"cpu": 43}
+
+def _resource_set_cpu_44(_os_name, _inputs):
+    return {"cpu": 44}
+
+def _resource_set_cpu_45(_os_name, _inputs):
+    return {"cpu": 45}
+
+def _resource_set_cpu_46(_os_name, _inputs):
+    return {"cpu": 46}
+
+def _resource_set_cpu_47(_os_name, _inputs):
+    return {"cpu": 47}
+
+def _resource_set_cpu_48(_os_name, _inputs):
+    return {"cpu": 48}
+
+def _resource_set_cpu_49(_os_name, _inputs):
+    return {"cpu": 49}
+
+def _resource_set_cpu_50(_os_name, _inputs):
+    return {"cpu": 50}
+
+def _resource_set_cpu_51(_os_name, _inputs):
+    return {"cpu": 51}
+
+def _resource_set_cpu_52(_os_name, _inputs):
+    return {"cpu": 52}
+
+def _resource_set_cpu_53(_os_name, _inputs):
+    return {"cpu": 53}
+
+def _resource_set_cpu_54(_os_name, _inputs):
+    return {"cpu": 54}
+
+def _resource_set_cpu_55(_os_name, _inputs):
+    return {"cpu": 55}
+
+def _resource_set_cpu_56(_os_name, _inputs):
+    return {"cpu": 56}
+
+def _resource_set_cpu_57(_os_name, _inputs):
+    return {"cpu": 57}
+
+def _resource_set_cpu_58(_os_name, _inputs):
+    return {"cpu": 58}
+
+def _resource_set_cpu_59(_os_name, _inputs):
+    return {"cpu": 59}
+
+def _resource_set_cpu_60(_os_name, _inputs):
+    return {"cpu": 60}
+
+def _resource_set_cpu_61(_os_name, _inputs):
+    return {"cpu": 61}
+
+def _resource_set_cpu_62(_os_name, _inputs):
+    return {"cpu": 62}
+
+def _resource_set_cpu_63(_os_name, _inputs):
+    return {"cpu": 63}
+
+def _resource_set_cpu_64(_os_name, _inputs):
+    return {"cpu": 64}
+
+_RESOURCE_SETS = {
```

Comment:
Hopefully all this can be deleted once https://github.com/bazelbuild/bazel/issues/15187 is addressed in some form.